app/backend/src/couchers/utils.py

```diff
@@ -44,6 +44,12 @@ def to_aware_datetime(ts: Timestamp):
 def now():
     return datetime.now(utc)
 
+def required_birthdate():
+    """
+    Most recent birthdate allowed to register (must be 18 years minimum)
+    """
+    dt = now().date()
```

Comment:
use `today_ = today()` (underscore to not overwrite the `today` object)

`dt` normally stands for datetime, but you only have a date